skyportal/handlers/api/source_groups.py

```diff
@@ -121,6 +128,22 @@ def post(self):
                 source.active = False
                 source.unsaved_at = datetime.datetime.utcnow()
 
+            if len(unsave_group_ids) > 0:
+                # Delete all this source auto-published pages, link to a release and sharing no groups with this source
```

Comment:
i think the sentence sounds a little odd. the comment could be shortened if not removed